vue3-apollo-quasar/public/mockServiceWorker.js

```diff
@@ -132,21 +132,21 @@ self.addEventListener('fetch', function (event) {
         request.method,
         request.url,
         `${error.name}: ${error.message}`,
-      );
+      )
     }),
-  );
-});
+  )
+})
```

Comment:
This has inconsistent styling with the rest of the codebase, please check your `.prettierrc` file